ngraph/python/BUILDING.md

```diff
@@ -1,76 +1,145 @@
 
-If you want to try a newer version of nGraph's Python API than is available
-from PyPI, you can build the latest version from source code. This process is
-very similar to what is outlined in our [ngraph_build] instructions with two
-important differences:
+## Build nGraph Python Wheels on Linux or MacOS
 
-1. You must specify: `-DNGRAPH_PYTHON_BUILD_ENABLE=ON` and `-DNGRAPH_ONNX_IMPORT_ENABLE=ON`
-   when running `cmake`.
+### Prerequisites
 
-2. Instead of running `make`, use the command `make python_wheel`.
+In order to build the nGraph Python wheel, you will need to install a few packages.
 
-    `$ cmake ../ -DNGRAPH_PYTHON_BUILD_ENABLE=ON -DNGRAPH_ONNX_IMPORT_ENABLE=ON`
+On Ubuntu 20.04 LTS you can use the following instructions to install the required packages, including Python and Cython.
 
-    `$ make python_wheel`
+    apt install git wget build-essential cmake
+    apt install python3 python3-dev python3-pip python-is-python3
+    pip3 install cython
 
-After this procedure completes, the `ngraph/build/python/dist` directory should
-contain the Python packages of the version you cloned. For example, if you
-checked out and built `0.21` for Python 3.7, you might see something like:
+You can see a full working example od an Ubuntu environment used in our continuous environment in this 
+[Dockerfile](https://github.com/openvinotoolkit/openvino/blob/master/.ci/openvino-onnx/Dockerfile).
 
-    $ ls python/dist/
-    ngraph-core-0.21.0rc0.tar.gz
-    ngraph_core-0.21.0rc0-cp37-cp37m-linux_x86_64.whl
+On MacOS you can use [Homebrew](https://brew.sh) to install required packages:
 
-## Building nGraph Python Wheels on Windows
+    brew install cmake
+    brew install automake
+    brew install libtool
 
-The build process on Windows consists of 3 steps:
+### Configure, build and install OpenVINO
+
+The following section will illustrate how to download, build and install OpenVINO in a workspace directory specified
+by the `${MY_OPENVINO_BASEDIR}` variable. Let's start by setting this variable to a directory of your choice: 
+
+    export MY_OPENVINO_BASEDIR=/path/to/my/workspace
+
+Now we can clone OpenVINO, configure it using `cmake` and build using `make`. Please note that we're disabling
+the building of a few modules by setting the `ENABLE_*` flag to `OFF`. In order to build the OpenVINO Python APIs
+set the mentioned flags to `ON`. Note the `CMAKE_INSTALL_PREFIX`, which defaults to `/usr/local/` if not set.
+
+    cd "${MY_OPENVINO_BASEDIR}"
+    git clone --recursive https://github.com/openvinotoolkit/openvino.git
+    mkdir openvino/build
+    cd openvino/build
+    
+    cmake .. \
+        -DENABLE_CLDNN=OFF \
+        -DENABLE_OPENCV=OFF \
+        -DENABLE_VPU=OFF \
+        -DENABLE_PYTHON=ON \
+        -DNGRAPH_PYTHON_BUILD_ENABLE=ON \
+        -DNGRAPH_ONNX_IMPORT_ENABLE=ON \
+        -DCMAKE_INSTALL_PREFIX="${MY_OPENVINO_BASEDIR}/openvino_dist"
```

Comment:
Minor: I think that it is worth to add that user can specify used version of Python by `-DPYTHON_EXECUTABLE=`which python3.8`.

Reply:
I just wanted to add the same comment as above - on MacOS (just testes) without specifying DPYTHON_EXECUTABLE, system uses python2 by default - and this is not supported version of python here